bedrock-devnet/devnet/__init__.py

```diff
@@ -160,15 +162,126 @@ def init_devnet_l1_deploy_config(paths, update_timestamp=False):
         deploy_config['l1GenesisBlockTimestamp'] = '{:#x}'.format(int(time.time()))
     write_json(paths.devnet_config_path, deploy_config)
 
-def devnet_l1_genesis(paths):
+def extDumpState(url: str, account: str):
```

Comment:
yeap, I did (As a side-note (and because we talked about it recently) I think this is why I prefer to make single comments; otherwise the reviews might get outdated before you submit the whole thing)

Reply:
Well, I am a fan of atomic reviews (for the first iteration at least) 😅. Closing conversation as it got resolved.